### ISSUE TITLE
Benchmark : XgBoost performance on fashion-mnist 89.8% and on MNIST 96.8%

### ISSUE THREAD

**Issue by anktplwl91** [zalandoresearch/fashion-mnist#88]
Hello, I tried XgBoost on both Fashion-MNIST and MNIST dataset, with the only pre-processing as scaling the pixel values to mean=0.0 and var=1.0.

**Fashion-MNIST**
Train accuracy 99.5%
Validation accuracy 90.7%
Test accuracy 89.8%

**MNIST**
Train accuracy 99.7%
Validation accuracy 97.4%
Test accuracy 96.8%

 **Notebook link**
https://github.com/anktplwl91/fashion_mnist.git

**Comment by CaptainVoid** [comment]
Isn't it wrong to do fit_transform() instead of transform() on test set while scaling?
